docs/deploy-development-rp.md

```diff
@@ -242,6 +242,12 @@ After that, when you [create](https://github.com/Azure/ARO-RP/blob/master/docs/d
   curl -X GET -k "https://localhost:8443/admin/subscriptions/$AZURE_SUBSCRIPTION_ID/resourceGroups/$RESOURCEGROUP/providers/Microsoft.RedHatOpenShift/openShiftClusters/$CLUSTER/kubernetespodlogs?podname=$POD&namespace=$NAMESPACE&container=$CONTAINER"
   ```
 
+* List Supported VM Sizes
+  ```bash
+  INSTANCETYPE=<master or worker>
```

Comment:
IMO we should not use the same term `instance type` to mean something else. Can we change this to `vmRole`, `nodeRole`, or something along these lines. The name change should cover any usage of this new configuration parameter?

Reply:
vmRole sounds more apt, made the suggested change.